frontend/packages/console-app/src/features.ts

```diff
@@ -0,0 +1,26 @@
+import { setFlag, handleError } from '@console/internal/actions/features';
+import { k8sList, K8sResourceKind } from '@console/internal/module/k8s';
+import { FeatureDetector } from '@console/plugin-sdk';
+import { MultiClusterHubModel } from './models/index';
+
+export const MCH_AVAILABILITY_FLAG = 'MULTICLUSTERHUB_AVAILABLITY';
+
+export const detectMCHAvailability: FeatureDetector = async (dispatch) => {
+  try {
+    const multiclusterHubs = await k8sList(MultiClusterHubModel);
```

Comment:
We probably want to key on the feature gate we plan to add in https://issues.redhat.com/browse/CONSOLE-2845. We'd only want to disable web terminal if the multicluster is enabled for console, which won't always be the case when ACM is installed.

@christianvogt